lib/elm-test.js

```diff
@@ -450,15 +448,9 @@ function generateAndRun(
       pathToElmBinary,
       args.report
     )
-      .then(function () {
-        return Runner.findTests(
-          elmVersion,
-          generatedCodeDir,
-          testFilePaths,
-          sourceDirs,
-          !Compile.isMachineReadableReporter(report)
-        )
-          .then(function (testModules) {
```

Comment:
What was this function call doing before?

Reply:
Ah I should have read the description first!

> - Fixed an oversight where elmi-to-json was called twice in a row.

I guess this was what it was doing? Nice catch :)

Reply:
Correct!